### PR TITLE
Intly 3772

### PR DESCRIPTION
**Pull request by damienomurchu**
### What

https://issues.jboss.org/browse/INTLY-3772 - add smoke tests to e2e installation test

#### Automated

Easiest way to verify - confirm e2e tests run by CI at end of PR are passing

#### Manual

```
make test/e2e
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -195,9 +195,7 @@ deploy/integreatly-installation-cr.yml: export SELF_SIGNED_CERTS := true
 deploy/integreatly-installation-cr.yml: export INSTALLATION_NAME := example-installation
 deploy/integreatly-installation-cr.yml: export INSTALLATION_TYPE := managed
 deploy/integreatly-installation-cr.yml:
-	@echo "masterURL = $(MASTER_URL), routingSubdomain = $(ROUTING_SUBDOMAIN), selfSignedCerts = $(SELF_SIGNED_CERTS)"
-	sed "s,MASTER_URL,$(MASTER_URL),g" deploy/crds/examples/integreatly-installation-cr.yaml | \
-	sed "s/ROUTING_SUBDOMAIN/$(ROUTING_SUBDOMAIN)/g" | \
+	@echo "selfSignedCerts = $(SELF_SIGNED_CERTS)"
```
Comment (damienomurchu) [inline Makefile]: Removing as intly-operator auto-discovers master url and routing sub domain now.

File: test/e2e/integreatly_test.go [hunk 2]
```diff
@@ -79,24 +83,7 @@ func integreatlyManagedTest(t *testing.T, f *framework.Framework, ctx *framework
...
-	}
-	err = f.Client.Get(goctx.TODO(), key, consoleRouteCR)
-	if err != nil {
-		return fmt.Errorf("could not get console route: %deploymentName", err)
-	}
-	masterURL := consoleRouteCR.Status.Ingress[0].Host
-	routingSubdomain := consoleRouteCR.Status.Ingress[0].RouterCanonicalHostname
-
-	t.Logf("Creating installation CR with routingSubdomain:%s, masterURL:%s\n", routingSubdomain, masterURL)
+	t.Logf("Creating installation CR\n")
```
Comment (damienomurchu) [inline test/e2e/integreatly_test.go]: Updating as removed logic is now in intly-operator

**Comment by damienomurchu** [comment]
Holding from being merged, as will version checks into this PR before its ready to be merged

/hold

File: test/e2e/integreatly_test.go [hunk 3]
```diff
@@ -184,9 +173,239 @@ func integreatlyManagedTest(t *testing.T, f *framework.Framework, ctx *framework
...
 		return err
 	}
+
+	// check namespaces labelled correctly
+	expectedNamespaces := []string{
+		"3scale",
+		"amq-online",
+		"codeready-workspaces",
+		"fuse",
+		"launcher",
```
Comment (philbrookes) [inline test/e2e/integreatly_test.go]: launcher is gone now

File: test/e2e/integreatly_test.go [hunk 4]
```diff
@@ -184,9 +173,239 @@ func integreatlyManagedTest(t *testing.T, f *framework.Framework, ctx *framework
...
+	}
+
+	// check products stage operator versions
+	stage = operator.StageName("products")
+	productOperators := map[string]string{
+		"3scale":                "1.9.8",
+		"amqonline":             string(operator.OperatorVersionAMQOnline),
+		"codeready-workspaces":  string(operator.OperatorVersionCodeReadyWorkspaces),
+		"fuse-on-openshift":     string(operator.OperatorVersionFuse),
+		"launcher":              string(operator.OperatorVersionLauncher),
```
Comment (philbrookes) [inline test/e2e/integreatly_test.go]: here too

**Comment by damienomurchu** [comment]
Removing hold now as all version check tests added. All tests should pass now too

/unhold

**Comment by damienomurchu** [comment]
@philbrookes refererences to launcher removed now, fyi

File: test/e2e/integreatly_test.go [hunk 5]
```diff
@@ -27,14 +30,15 @@ import (
 
 var (
 	retryInterval                    = time.Second * 5
-	timeout                          = time.Second * 60
+	timeout                          = time.Second * 75
```
Comment (damienomurchu) [inline test/e2e/integreatly_test.go]: this isn't really needed, and will remove in another PR so as not to trigger the e2e-spiral-of-doom :D

**Pull request merged**